editor/src/core/shared/file-utils.ts

```diff
@@ -78,48 +89,89 @@ export function extractText(file: File): Promise<TextResult> {
   })
 }
 
-export function extractAsset(file: File): Promise<AssetResult> {
-  return new Promise((resolve, reject) => {
+export async function extractAsset(file: File): Promise<AssetResult> {
```

Comment:
why make it async? are you using `await` inside this function body? or to bring it in line with some other function's signature?

Reply:
It was already async because it returned a `Promise`, adding that got rid of an eslint warning.